develop/api_guc.rst

```diff
@@ -395,6 +450,31 @@ citus.partition_buffer_size (integer)
 
 Sets the buffer size to use for partition operations and defaults to 8MB. Citus allows for table data to be re-partitioned into multiple files when two large tables are being joined. After this partition buffer fills up, the repartitioned data is flushed into files on disk. This configuration entry can be set at run-time and is effective on the workers.
 
+Real-time executor configuration (deprecated)
```

Comment:
Is there still a use for the real-time executor? Assuming that if people want that behavior they can use the adaptive executor with `force_max_query_parallelization`